sdk/tests/deqp/functional/gles3/multisample/00_test_list.txt

```diff
@@ -0,0 +1,4 @@
+default_fbo.html
```

Comment:
nit: Autogenerated file warning, if comments are allowed in 00_test_list.txt

Reply:
Thanks, added.